Resources/Prototypes/Research/civilianservices.yml

```diff
@@ -8,10 +8,11 @@
     state: seedextractor
   discipline: CivilianServices
   tier: 1
-  cost: 5000
+  cost: 5500
```

Comment:
extra 500 points may as well not exist + no deltav comment saying it used to be 5000
bring to 7500 imo

Reply:
changes made~